tests/cli/test_workflow.py

```diff
@@ -1135,3 +1136,34 @@ def test_workflow_execute_docker_toil_stderr(runner, client, run_shell):
 
     assert 1 == result.exit_code, format_result_exception(result)
     assert "Cannot run workflows that have stdin or stderr redirection with Docker" in result.output
+
+
+@pytest.mark.parametrize("provider", available_workflow_providers())
+@pytest.mark.parametrize(
+    "workflow, parameters, outputs",
+    [
+        (
+            [
+                "--output {a-2}_{b-3} python -c \"import sys; f=open(sys.argv[2]+'_'+sys.argv[4], 'w'); f.write('foo'); f.close()\" -a foo -b bar",
```

Comment:
haven't really had a better idea for what could be tested with `execute -s`. open for suggestions